licenses/license-engine.sh

```diff
@@ -48,6 +48,9 @@ ALLOWED_LICENSES=(
 	'(WTFPL OR MIT)'
 	'(MIT OR WTFPL)'
 	'LGPL-3.0-or-later' # Requires only that modifications to LGPL-protected libraries are published under a GPL-compatible license which is not the case at Ory
+	'Apache-2.0 AND MIT'
+	'(Apache-2.0 OR MPL-1.1)'
```

Comment:
Not sure if this is okay. We pull in a dep that uses this dual license model, through an MIT licensed module. 

```
└─┬ @nx/jest@20.1.0
  └─┬ identity-obj-proxy@3.0.0
    └── harmony-reflect@1.6.2
```

(@nx/jest is MIT licensed: https://www.npmjs.com/package/@nx/jest)
So I think this is fine!

Reply:
This is known as dual licensing and as long as it's OR it should be fine:

> Dual licensing offers two distinct license options. When a project uses something like (Apache-2.0 OR MPL-1.1), you pick one of them and follow its terms. If you distribute the software under Apache-2.0, you comply with Apache-2.0 requirements. If you choose MPL-1.1, you comply with MPL-1.1 requirements.